src/service/resources-impl.js

```diff
@@ -784,7 +794,10 @@ export class Resources {
             minTop = minTop == -1 ? box.top : Math.min(minTop, box.top);
           }
           request.resource./*OK*/changeSize(
-              request.newHeight, request.newWidth, request.callback);
+              request.newHeight, request.newWidth);
+          if (request.callback) {
+            request.callback(/* hasSizeChanged */true);
+          }
           request.resource.overflowCallback(/* overflown */ false,
               request.newHeight, request.newWidth);
```

Comment:
get rid of the params - height and width here - and all the way till the end , as this won't be needed anywhere